docs/search/behavior-insights/logreduce/index.md

```diff
@@ -32,25 +32,25 @@ In this section, we'll introduce the following concepts:
 <div className="box-wrapper" >
```

Comment:
The cards need to be updated to show the current articles in the section.

Reply:
great catch!